packages/oui-dual-list/src/dual-list.controller.js

```diff
@@ -0,0 +1,247 @@
+export default class {
+    constructor ($q, $element, ouiDualListProvider) {
+        "ngInject";
+
+        this.$q = $q;
+        this.$element = $element;
+        this.dualListProvider = ouiDualListProvider;
+    }
+
+    $onInit () {
+        this.sourceListLabel = this.sourceListLabel || this.dualListProvider.translations.sourceListLabel;
+        this.targetListLabel = this.targetListLabel || this.dualListProvider.translations.targetListLabel;
+        this.moveAllLabel = this.moveAllLabel || this.dualListProvider.translations.moveAllLabel;
+        this.removeAllLabel = this.removeAllLabel || this.dualListProvider.translations.removeAllLabel;
+        this.sourceListEmptyLabel = this.sourceListEmptyLabel || this.dualListProvider.translations.sourceListEmptyLabel;
+        this.targetListEmptyLabel = this.targetListEmptyLabel || this.dualListProvider.translations.targetListEmptyLabel;
+        this.addLabel = this.addLabel || this.dualListProvider.translations.addLabel;
+        this.bulkActionEnabled = this.bulkActionEnabled || this.dualListProvider.bulkActionEnabled;
+        this.sourceList = this.sourceList || [];
+        this.targetList = this.targetList || [];
+        this.onAdd = this.onAdd || null;
+        this.onRemove = this.onRemove || null;
+        this.property = this.property || null;
+
+        this.loadingMap = {};
+        this.targetCloseIconStyle = {};
+        this.targetOpenIconStyle = {};
+        this.targetContentStyle = {};
+        this.sourceCloseIconStyle = {};
+        this.sourceOpenIconStyle = {};
+        this.sourceContentStyle = {};
+        this.sourceListLoading = false;
+        this.targetListLoading = false;
+        this.displayNoneStyle = { display: "none" };
```

Comment:
No inline style in our JS. Use CSS classes.
Or use `ngShow`, `ngHide` or `ngIf` if necessary :)